### ISSUE TITLE
Bug: ApiException on RolesApi.GetPermission and RolesApi.GetPermissionAsync

### ISSUE THREAD

**Issue by infolabnz** [kinde-oss/kinde-dotnet-sdk#21]
### Prerequisites

- [X] I have searched the repository’s issues and [Kinde community](https://thekindecommunity.slack.com/archives/C04K316BXEH) to ensure my issue isn’t a duplicate
- [X] I have checked the latest version of the library to replicate my issue
- [X] I have read the [contributing guidelines](https://github.com/kinde-oss/.github/blob/main/.github/CONTRIBUTING.md)
- [X] I agree to the terms within the [code of conduct](https://github.com/kinde-oss/.github/blob/main/.github/CODE_OF_CONDUCT.md)

### Describe the issue

```
Kinde.Api.Client.ApiException: 'Cannot deserialize the current JSON object (e.g. {"name":"value"}) into type 'System.Collections.Generic.List`1[Kinde.Api.Model.RolesPermissionResponseInner]' because the type requires a JSON array (e.g. [1,2,3]) to deserialize correctly.
To fix this error either change the JSON to a JSON array (e.g. [1,2,3]) or change the deserialized type so that it is a normal .NET type (e.g. not a primitive type like integer, not a collection type like an array or List<T>) that can be deserialized from a JSON object. JsonObjectAttribute can also be added to the type to force it to deserialize from a JSON object.
Path 'code', line 1, position 8.'
```

Also, the naming of this end point should be `GetRolePermissionsAsync` (plural).

### Library URL

https://github.com/kinde-oss/kinde-dotnet-sdk

### Library version

1.2.8

### Operating system(s)

Windows

### Operating system version(s)

Wwindows 11 v10.0.22635

**Comment by coel** [comment]
Thank you for raising this, I will look into it.

**Comment by coel** [comment]
Hi @infolabnz , we just released a new version `1.2.9`, can you update and see if these are resolved?

**Comment by infolabnz** [comment]
Great, thank you. Yes, that has been resolved. Thanks for changing the method name, too. That makes much more sense.
